mergin/client.py

```diff
@@ -568,6 +570,7 @@ def _do_request(self, request):
 
             if self._auth_session:
                 request.add_header("Authorization", self._auth_session["token"])
+                request.add_header("User-Agent", platform.system() + "-" + platform.release() + "/" + self.client_version)
```

Comment:
I think we wanted it the other way around: "client/version (platform/version)"

also instead of platform.release() we could try to get linux distribution or mac_os/windows version depending on system